include/library/spdm_requester_lib.h

```diff
@@ -24,6 +24,128 @@
  **/
 libspdm_return_t libspdm_init_connection(void *spdm_context, bool get_version_only);
 
+/**
+ * This function builds GET_VERSION request message.
+ *
+ * @param  spdm_context      A pointer to the SPDM context.
+ * @param  request           request messge buffer.
+ * @param  request_size      On input, indicates the size in bytes of request message buffer.
+ *                           On outout, indicates the size in bytes of the request message.
+ *
+ * @retval LIBSPDM_STATUS_SUCCESS   The request message is created in the buffer.
+ **/
+libspdm_return_t
+libspdm_build_request_get_version(
```

Comment:
`libspdm_return_t libspdm_*` should be on one line. There are many instances of this.

Reply:
Thanks. I have updated the style.